src/Microsoft.Android.Templates/android-bindinglib/AndroidBinding1.csproj

```diff
@@ -3,5 +3,7 @@
     <TargetFramework>net6.0-android</TargetFramework>
     <SupportedOSPlatformVersion>SUPPORTED_OS_PLATFORM_VERSION</SupportedOSPlatformVersion>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">AndroidBinding1</RootNamespace>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Should we update `AndroidBinding1.csproj` to *also* set `$(AndroidGenerateResourceDesigner)`=False, as per #6224?